lib/generators/rolemodel/soft_destroyable/templates/soft_destroyable.rb

```diff
@@ -70,9 +70,20 @@ def restore!
     restore(deleted_at) || raise_validation_errors
   end
 
+  def cascade_soft_destroy_nullify
```

Comment:
Hmm, this would come in handy for a specific case in Gimme Credit https://github.com/RoleModel/gimme-credit/pull/495/files

Is there a reason you aren't checking `belongs_to` and `has_one` associations?

@BlaineIrvin perhaps we should use this pattern in Gimme Credit?

Reply:
@justwiebe No, the `has_one` would be another good one to include! 

I'm curious why we'd need to do anything to our `belongs_to`, could you explain that one?

Reply:
`belongs_to` might be an even bigger edge case, but for our situation, we've got a chain of reports that "replace" the previous one. We disallow soft destroying a report in the middle of the chain to side step complications, but we do allow soft destroying the latest report. To keep the chain linear, we decided to remove the soft destroyed one from it altogether be removing the `belongs_to` association

Reply:
It sounds like you're having the child class nullify its own association. There could CERTAINLY be value in that one!

Reply:
That sounds like an `after_soft_destroy nullify_self`

Reply:
Well sure, maybe not out of the box. You could use an arbitrary value on that dependent call. 😉

Reply:
@justwiebe @BlaineIrvin Thinking back on this - I'm going to add the `has_one` update. I think the specific case of the parent record might be best solved by touching the parent record.